addons/event_booth/views/event_event_views.xml

```diff
@@ -21,16 +21,6 @@
                     </div>
                 </button>
             </div>
-            <page name="tickets" position="after">
```

Comment:
Why that removal ? Was a fix done that allow it to work ? I remember we tried to remove it when developing but it was not working as intended ...

Reply:
I don't know about a fix. But I was indeed surprised that it worked by removing the field from the view...